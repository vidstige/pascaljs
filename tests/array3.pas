program array3;
type
    TMAtrix3 = record
        m: array[0..9] of Real;
    end;
var
  matrix: TMAtrix3;
begin
  matrix.m[1] := 1.5;
  WriteLn('matrix.m[1] = ', matrix.m[1]);
end.