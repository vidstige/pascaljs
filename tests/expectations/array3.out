matrix.m[1] = 1.5
